lib/vector-buffers/src/variants/disk_v2/record.rs

```diff
@@ -20,7 +20,7 @@ pub enum RecordStatus {
     /// The record was able to be read from the buffer, and the checksum is valid.
     ///
     /// Contains the ID for the given record, as well as the metadata.
-    Valid { id: u64, metadata: u32 },
```

Comment:
cc/ @tobz just to double check this one. I'm wondering if we meant to be using it somewhere.

Reply:
Taking a look at it, I think it might have just been me optimizing/trying to anticipate a need that never came to pass.

If the code compiles and works without it, that's good enough for me.